en/flight_controller/kakutef7.md

```diff
@@ -0,0 +1,158 @@
+| M5, M6 | Additional motor signal outputs (located at side of board) ||
+| RSI | Analog RSSI (0-3.3v) input from receiver ||
+| Boot | Bootloader button ||
+
+
+TODO: parts of the following is duplicated from the Omnibus board.
+
+## PX4 Bootloader Update {#upload}
+
+The board comes pre-installed with [Betaflight](https://github.com/betaflight/betaflight/wiki). 
+Before PX4 firmware can be installed, the *PX4 bootloader* must be flashed.
+
+There are two options for flashing the bootloader: via *Betaflight Configurator* (easier), or building from source (guaranteed to work).
+
+### Bootloader Update using Betaflight Configurator {#betaflight_configurator}
+
+To install the PX4 bootloader using the *Betaflight Configurator*:
+1. Download the pre-built bootloader binary: [omnibusf4sd_bl.hex](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/omnibus_f4_sd/omnibusf4sd_bl_d52b70cb39.hex).
+1. Download the [Betaflight Configurator](https://github.com/betaflight/betaflight-configurator/releases) for your platform.
+   > **Tip** If using the *Chrome* web browser, a simple cross-platform alternative is to install the configurator as an [extension from here]( https://chrome.google.com/webstore/detail/betaflight-configurator/kdaghagfopacdngbohiknlhcocjccjao). 
+1. Connect the board to your PC and start the Configurator.
+1. Press the **Load Firmware [Local]** button
+   ![Betaflight Configurator - Local Firmware](../../assets/flight_controller/omnibus_f4_sd/betaflight_configurator.jpg)
+1. Select the bootloader binary from the file system and then flash the board.
+
+You should now be able to install PX4 firmware on the board.
+
+### Bootloader Update using Source
+
+#### Download Bootloader Source
+
+Download and build the [Bootloader](https://github.com/PX4/Bootloader) via:
+```
+git clone --recursive  https://github.com/PX4/Bootloader.git
+cd Bootloader
+make omnibusf4sd_bl
+```
+
+#### Flash Bootloader
+
+You can flash the PX4 bootloader using the [dfu-util](http://dfu-util.sourceforge.net/) or the graphical [dfuse](https://www.st.com/en/development-tools/stsw-stm32080.html) tool on windows.
+
+Don't be afraid to try flashing using any of the below methods. 
+The STM32 MCU cannot be bricked. 
+DFU cannot be overwritten by flashing and will always allow you to install a new firmware, even if flashing fails.
+
+##### Enter DFU mode
+
+Both methods require the board to be in DFU mode. 
+To enter DFU mode, hold the boot button down while connecting the USB cable to your computer. 
+The button can be released after the board is powered up.
+
+##### dfu-util
+
+```
+dfu-util -a 0 --dfuse-address 0x08000000 -D  build/omnibusf4sd_bl/omnibusf4sd_bl.bin
+```
+
+Reboot the flight controller and it let it boot without holding the boot button.
+
+##### dfuse
+
+See the dfuse manual is here: https://www.st.com/resource/en/user_manual/cd00155676.pdf
+
+Flash the `omnibusf4sd_bl.bin` file.
```

Comment:
FWIW I tend to use bold for file names, code for terminal commands. This is fine if you have been consistent across doc though.